pehlaschoollibrary/Classes/BookView.cpp

```diff
@@ -442,19 +440,23 @@ void BookView::nextPage()
                                                     LogManager::getInstance()->logEvent(
                                                             _book->bookTitle, "finish_read", "",
                                                             _currentPage);
-                                                    CompletePopup::create()->show(0.0, [this]() {
+                                                    CompletePopup::create()->show(0, [this]() {
                                                         _isReadAll = true;
-                                                        
                                                         if (_checkCompleteCondition)
                                                         {
-                                                            CCAppController::sharedAppController()->handleGameComplete(1);
+                                                           //Finish the activity, return back to the MainActivity and restart the story
```

Comment:
Comment should be more user oriented. For e.g. Finish the current activity and go back to XYZ screen.